### DISCUSSION TITLE
SocketException: Failed to create server socket (port = 53318)

### DISCUSSION THREAD

**Discussion by aagha** [localsend/localsend#2120] [Discussion (English, International)]
Starting today with 1.16.1 (56), I'm getting this error as soon as I launch LocalSend:

![image](https://github.com/user-attachments/assets/2346b5b2-815f-49aa-9fc1-b416a6951a84)

![image](https://github.com/user-attachments/assets/b8a76a6f-8bca-48ef-a733-0173e74e70ae)

Any idea why?

**Comment by Tienisto** [comment]
Can you check if the network is configured as a "private" network?
